client/src/__helpers__/Constants.js

```diff
@@ -8,4 +8,17 @@ export const LoginConstants = {
         minLength: 8,
         maxLength: 24
     }
+}
+
+// Constants related to toast alerts
+export const ToastConstants = {
+    loginErrorMsg: {
+        bothFieldsFilled: "The username or password you have entered is invalid or not recognized. Try again.",
```

Comment:
Key should be named specific to the action.  This would be an "unathorizedLogin"

Reply:
Done